plugins/kubernetes.yaml

```diff
@@ -158,9 +151,39 @@
       - id: kubelet_filter_router
         type: router
         routes:
-          - output: add_kublet_metadata
+          - output: kubelet_message_parser_router
             expr: '$record._SYSTEMD_UNIT == "kubelet.service"'
 
+      - id: kubelet_message_parser_router
+        type: router
+        routes:
+          - output: message_regex_parser
+            expr: '$record.MESSAGE matches "^\\w\\d{4}"'
+          - output: add_kublet_metadata
+            expr: true
+
+      - id: message_regex_parser
+        type: regex_parser
+        parse_from: MESSAGE
+        regex: '(?P<severity>\w)(?P<timestamp>\d{4} \d{2}:\d{2}:\d{2}.\d+)\s+(?P<pid>\d+)\s+(?P<source>[^ \]]+)\] (?P<message>.*)'
+        severity:
+          parse_from: severity
+          mapping:
+            debug:
+              - d
```

Comment:
This is minor, but just fyi - you can actually specify just a single value rather than a list. It might be worth doing for readability.

```
mapping:
  debug: d
  info: i
  warning: w
  error: e
  critical: c
```

Reply:
I like it 👍